pdh_check/datadog_checks/pdh_check/data/conf.yaml.example

```
@@ -12,7 +28,7 @@ instances:
     ## @param countersetname - string - required
     ## `countersetname` is the name of the performance counter
     #
-  - countersetname: <PERFORMANCE_COUNTER_NAME>
+    - countersetname: <PERFORMANCE_COUNTER_NAME>
```

The hyphen doesn't count in YAML indentation. This makes the yaml library unable to read the file.
```suggestion
  - countersetname: <PERFORMANCE_COUNTER_NAME>
```